core/sql/executor/ExExeUtilGet.cpp

```diff
@@ -3173,6 +3149,21 @@ short ExExeUtilGetHbaseObjectsTcb::work()
           }
           break;
 
+        case EVAL_EXPR_:
```

Comment:
This is a very nice feature. Thanks for adding this. I've always wondered though why GET statements aren't simply SELECTs. Would be nice to embed them within other SELECTs for example. There have been times when I've wanted to join them to other things too.